.github/workflows/template_gitops.yml

```diff
@@ -128,4 +131,7 @@ jobs:
           gitops-dev: ${{ inputs.gitops-dev }}
           gitops-stage: ${{ inputs.gitops-stage }}
           gitops-prod: ${{ inputs.gitops-prod }}
+          upwind-client-id: ${{ vars.UPWIND_CLIENT_ID }}
+          upwind-client-secret: ${{ secrets.upwind-client-secret }}
+          upwind-organization-id: ${{ vars.UPWIND_ORGANIZATION_ID }}
```

Comment:
I don't know if it is a good idea to use the context variables in reusable workflows?! 🤔 It would be the first workflow and I think it is better to make the needed variables explicitly. WDYT?

Reply:
@0x46616c6b agrees that we should do it your way *for now*. So, I'll update it in a bit.